pkg/subctl/cmd/root.go

```diff
@@ -64,23 +64,27 @@ func Execute() error {
 func init() {
 	rootCmd.AddCommand(cmdversion.Cmd)
 	cloudCmd := cloud.NewCommand(&kubeConfig, &kubeContext)
-	addKubeContextFlag(cloudCmd)
+	AddKubeContextFlag(cloudCmd)
 	rootCmd.AddCommand(cloudCmd)
 }
 
-func addKubeConfigFlag(cmd *cobra.Command) {
+func AddToRootCommand(cmd *cobra.Command) {
```

Comment:
I'd argue that it's better if the "higher level" command add it's options and not vice versa.

It makes it easier to maintain and understand the code, and it's more in line with how you actually discover the commands (e.g. when you run `subctl` you get a list of it's sub commands).

WDYT?

Reply:
Unfortunately that would create a cyclic package dependency.